_api-reference/script-apis/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Script APIs
+has_children: true
+nav_order: 70
+redirect_from:
+  - /opensearch/rest-api/script-apis/
+---
+
+# Script APIs
+
+The script APIs allow you to work with stored scripts. Stored scripts are part of the cluster state and reduce compilation time and enhance search speed. The default scripting language is Painless.
+
+You perform the following operations on stored scripts:
```

Comment:
I think if the links matched the destination headings we could simplify these lines a little bit.  Something like:

```
You can perform the following operations on stored scripts:
* [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/)
* [Execute Painless stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/exec-stored-script/)
* [Get stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/get-stored-script/)
* [Delete script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/delete-script/)
* [Get stored script contexts]({{site.url}}{{site.baseurl}}/api-reference/script-apis/get-script-contexts/).
* [Get script language]({{site.url}}{{site.baseurl}}/api-reference/script-apis/get-script-language/)
```